PORTAL/jobs/_workers.py

```diff
@@ -121,7 +122,7 @@ class Workers:
     """The set of configured workers."""
 
     @classmethod
-    def from_config(cls, cfg, JobsFS=WorkerJobsFS):
+    def from_config(cls, cfg: JobsConfig, JobsFS=WorkerJobsFS) -> Workers:
         worker = Worker.from_config(cfg.worker, JobsFS)
```

Comment:
For this, `Optional` doesn't work because the type checker notices that there are no None-checks inside the function.  This can be set to `Type[WorkerJobsFS]` which would accept `WorkerJobsFS` or any subclass.